app/main.py

```diff
@@ -1,13 +1,23 @@
 def format_linter_error(error: dict) -> dict:
-    # write your code here
-    pass
+    return {
+        "line": error.get("line_number"),
+        "column": error.get("column_number"),
+        "message": error.get("text"),
+        "name": error.get("code"),
+        "source": "flake8"
+
+    }
 
 
 def format_single_linter_file(file_path: str, errors: list) -> dict:
-    # write your code here
-    pass
+    return {
```

Comment:
the output sequence should be as follows
<img width="883" alt="Screenshot 2023-07-21 at 16 35 44" src="https://github.com/mate-academy/py-linter-formatter/assets/117740672/026b169a-e3e7-4b05-9531-14708e26e24a">